watch-package.js

```diff
@@ -8,16 +8,24 @@ var through = require('through2')
 var npm = process.platform === 'win32' ? 'npm.cmd' : 'npm';
 var nodemon = process.platform === 'win32' ? 'nodemon.cmd' : 'nodemon';
 
-module.exports = function watchPackage(pkgDir, exit) {
+var pkgDir = '';
+var stdin = null;
+
+module.exports = function watchPackage(_pkgDir, exit, taskName) {
```

Comment:
You are right it would be possible if anyone would call it "npm-watch '    '".
Not thought about it and will add the trim.